packages/Webkul/API/Http/Resources/Catalog/Product.php

```diff
@@ -56,6 +58,8 @@ public function toArray($request)
                     $product->getTypeInstance()->haveSpecialPrice(),
                     core()->currency($product->getTypeInstance()->getSpecialPrice())
                 ),
+            'regular_price'          => data_get($prices, 'regular_price.price'),
+            'formated_regular_price' => data_get($prices, 'regular_price.formated_price'),
```

Comment:
Hi, One more thing i want to add on. The regular price is the need when the product is having special price because if product is not having special price then user can automatically fetch the regular price from the price key.

So, just add `$this->when()` method to both key. You can take reference from the `special_price` and `formated_special_price`.